ios/Handlers/RNLongPressHandler.m

```diff
@@ -19,28 +19,25 @@ - (id)initWithGestureHandler:(RNGestureHandler*)gestureHandler;
 @end
 
 @implementation RNBetterLongPressGestureRecognizer {
-    __weak RNGestureHandler *_gestureHandler;
+  __weak RNGestureHandler *_gestureHandler;
 }
 
 - (id)initWithGestureHandler:(RNGestureHandler*)gestureHandler
 {
-    if ((self = [super initWithTarget:gestureHandler action:@selector(handleGesture:)])) {
-        _gestureHandler = gestureHandler;
-    }
-    return self;
+  if ((self = [super initWithTarget:gestureHandler action:@selector(handleGesture:)])) {
+    _gestureHandler = gestureHandler;
+  }
+  return self;
 }
 
 - (void)touchesMoved:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event
 {
-    [super touchesMoved:touches withEvent:event];
-
-    if (_gestureHandler.shouldCancelWhenOutside) {
-        CGPoint pt = [self locationInView:self.view];
-        if (!CGRectContainsPoint(self.view.bounds, pt)) {
-            self.enabled = NO;
-            self.enabled = YES;
-        }
-    }
+  [super touchesMoved:touches withEvent:event];
+  CGPoint pt = [self locationInView:self.view];
```

Comment:
why this change? It does not seem to change anything? If it is needed than it could be because of some side effect which in such a case should be properly documented